src/ucp/wireup/select.c

```diff
@@ -437,7 +509,8 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_select_transport(
     UCS_BITMAP_CLEAR(&addr_index_map);
     ucp_unpacked_address_for_each(ae, address) {
         addr_index = ucp_unpacked_address_index(address, ae);
-        if (!(remote_dev_bitmap & UCS_BIT(ae->dev_index))) {
+        if (ucp_wireup_remote_device_count_exceeded(remote_dev_count, ae)) {
+            /* exclude remote device if reached max concurrency level */
```

Comment:
`Exclude`
`max concurrency level is reached`

Reply:
reverted